addons-l10n/en/disable-auto-save.json

```
@@ -1,3 +1,3 @@
 {
-  "disable-auto-save/save-and-leave": "Message by \"disable auto-save\" addon from Scratch Addons: save changes and leave editor?"
+  "disable-auto-save/save-and-leave": "Message by \"disable auto-save\" addon from Scratch Addons: going back to the project page will save your changes. Do you want to save the project and leave the editor?"
```

```suggestion
  "disable-auto-save/save-and-leave": "Message by \"disable auto-save\" addon from Scratch Addons: going to the project page will save your changes. Do you want to save the project and leave the editor?"
```
The editor might have been opened directly (using the Create button or from My Stuff).